frontend/packages/dev-console/src/components/add/GettingStartedSection.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import {
+  GettingStartedGrid,
+  useGettingStartedShowState,
+  GettingStartedShowState,
+  QuickStartGettingStartedCard,
+} from '@console/shared/src/components/getting-started';
```

Comment:
```suggestion
} from '@console/shared';
```

Reply:
I would like to stay with this import as most imports are using this schema at the moment.